newIDE/app/package.json

```diff
@@ -21,6 +21,7 @@
     "classnames": "2.2.5",
     "create-react-context": "^0.1.6",
     "date-fns": "^1.29.0",
+    "electron": "^2.0.2",
```

Comment:
You should not commit any changes that you made to package.json or package-lock.json. Can you try to remove these? 
You may want to read about `git add` (if you're using command line) (and `git add -p`, which is super useful to choose the part to add in your commit). If you're using a graphical client you should also be able to choose the files to include in your commits.

In general, only commit the changes related to your feature - otherwise I'll have to edit your commits and remove these :)

Reply:
ah sorry, i thought it was in the gitignore

Reply:
The gitignore contains the file that are not in the git repository at all.
It's a bit a pain at the beginning but once you know how to choose the files to commit, you can make much cleaner commits :) (by not selecting files like package.json or package-lock.json, or files with irrelevant changes)